src/components/TabList/TabList.scss

```
@@ -0,0 +1,29 @@
+[class^='selectBtn'] {
+  width: calc(100% / 3);
+  display: flex;
+  align-items: center;
+  background-color: gray;
+  border-radius: 10px 10px 0 0;
+  padding: 13px;
+  color: #fff;
+  &.blue.active {
+    box-shadow: #329be7 0 4px inset;
+    background-color: #fff;
+    color: #000;
+  }
+  &.green.active {
+    box-shadow: #a3e743 0 4px inset;
+    background-color: #fff;
+    color: #000;
+  }
+  &.red.active {
+    box-shadow: #ff7d3a 0 4px inset;
+    background-color: #fff;
+    color: #000;
+  }
+  & img {
+    margin-right: 5px;
+    width: 18px;
+    height: 18px;
+  }
+}
```

active인 경우에만 color를 입히는 건가요? 이렇게 바꿔볼 수 있을 것 같습니다.
```suggestion
.selectBtn {
  width: calc(100% / 3);
  display: flex;
  align-items: center;
  background-color: gray;
  border-radius: 10px 10px 0 0;
  padding: 13px;
  color: #fff;
  
  &.active {
    background-color: #fff;
    color: #000;
    
    &.blue {
      box-shadow: #329be7 0 4px inset;
    }
    
    &.green {
      box-shadow: #a3e743 0 4px inset;
    }
    
    &.red {
      box-shadow: #ff7d3a 0 4px inset;
    }
  }
}
```